wordpress-mobile/WordPress-Android
# Issue/13573 add organization id to reader

Fixes #13573

This PR introduces the concept of Organization instead of "wp for teams" flag.
I replaced all the instances of flag usage with Organization check in the Reader.

I left the migration script that creates the `is_wp_for_teams` and `is_wpforteams_site` flags since I'm not sure if it's ok to change those?

To test:
- Test DB migration by installing this branch over develop.
- Check a8c tab in the reader and confirm that the special p2 header is visible at postcards from supported sites.

PS: Note that since it's a new field in DB table, you might need to refresh posts to populate it.

PR submission checklist:
